src/app/src/components/FilterSidebarGuideTab.jsx

```diff
@@ -1,44 +1,132 @@
+                The Open Apparel Registry (OAR) is an open data tool for
+                identifying global apparel facilities and their affiliations
+                (“Contributors”). The OAR brings together facility lists from
+                industry stakeholders into one map and registry and assigns each
+                unique facility an OAR ID.
+            </p>
+            <p className="control-panel__body">How does it work?</p>
+            <ol>
+                <li>
+                    With a{' '}
+                    <Link to={authRegisterFormRoute} href={authRegisterFormRoute}>
+                        Free Account
+                    </Link>
+                    , industry stakeholders can add a list of facilities, from
+                    any tier of the supply chain (excluding raw materials - so
+                    no cotton farms or cattle ranches, for example)
+                </li>
+                <li>
+                    The OAR software checks the uploaded list against facilities
+                    already in the system and uses the name and address to
+                    either match the facility with one already in the system, or
+                    to create a new facility on the map
+                </li>
+                <li>
+                    The user who added the list reviews the suggested matches
+                    and new listings and, in some instances where the algorithm
+                    cannot automatically make a decision, will be required to
+                    manually confirm or reject the options
+                </li>
+                <li>
+                    New, confirmed facilities are given their own unique OAR ID
+                    - a valuable tool for linking facilities across software
+                    systems and databases - and are added to the map
+                </li>
+                <li>
+                    Facilities that match with an entry already in the OAR
+                    database are updated to include the association with the new
+                    organization (the “Contributor”)
+                </li>
+            </ol>
+            <p className="control-panel__body">
+                Anybody can search the facilities and view associated
+                affiliations. Users can search the site by entering a facility
+                name or OAR ID number; searching by a particular contributor (or
+                multiple contributors); or searching by country (or multiple
+                countries). All of these search filters can be used in
+                combination.
+            </p>
+            <p className="control-panel__body">
+                Users can download the data for free by clicking the “Download”
+                button (free account required) or{' '}
+                <a href="http://openapparel.org/api/docs/">
+                    connecting via API
+                </a>
+                .
+            </p>
+            <p>
```

Comment:
This tag is missing `className="control-panel__body"`

Reply:
Fixed in a478d20